### DISCUSSION TITLE
Issue with npx hardhat verify on Sepolia

### DISCUSSION THREAD

**Discussion by PaladinDotEth** [NomicFoundation/hardhat#5936] [Troubleshooting]
I'm running into an issue when trying to verify a contract on the Sepolia network using npx hardhat verify. The error message states:

"An unexpected error occurred during the verification process. Please report this issue to the Hardhat team. Error Details: Invalid API Key"

I've double-checked my Etherscan API key, and everything seems to be in order on their end, so I'm not sure why this error keeps coming up. Has anyone else experienced this issue or have any suggestions on how to resolve it?

**Comment by kanej** [comment]
We have had other reports of issues with verifying against Sepolia. It was a temporary Etherscan issue. Can you retry?
